### ISSUE TITLE
dnscrypt-proxy 2

### ISSUE THREAD

**Issue by jedisct1** [alterstep/dnscrypt-osxclient#162]
If you are still alive, the first beta of dnscrypt-proxy 2.0.0 is out, and does everything your client needs, and more.

There's a catch, though: the configuration file is radically different.

**Comment by JayBrown** [comment]
Oh yes, please! v2 would be great! :)

**Comment by JayBrown** [comment]
For the time being, if you need to control the v2 proxy from the GUI, you can use one of these BitBar plugins:

https://github.com/jedisct1/bitbar-dnscrypt-proxy-switcher
https://github.com/JayBrown/DNSCrypt-Menu

**Comment by jedisct1** [comment]
Right, the combination of dnscrypt-proxy 2 always running as a background process + Bitbar to control it works very well.

It may be a bit complicated for most users, though.

I wish we had something like DNSCloak on macOS, but I don't have time to work on this.

**Comment by JayBrown** [comment]
Maybe something for DNSCloak's developer, once Marzipan is available?
